src/components/App.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react";
+import LocationDetails from "./LocationDetails";
+import ForecastSummaries from "./ForecastSummaries";
+import ForecastDetails from "./ForecastDetails";
+import getForecast from "../requests/getForecast";
+import SearchForm from "./SearchForm";
+import "../styles/App.css";
+
+const App = () => {
+  const [forecasts, setForecasts] = useState([]);
+  const [location, setLocation] = useState({ city: "", country: "" });
+  const [selectedDate, setSelectedDate] = useState(0);
+  const [searchText, setSearchText] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+  const selectedForecast = forecasts.find(
+    (forecast) => forecast.date === selectedDate
+  );
+
+  useEffect(() => {
+    getForecast(
+      setSelectedDate,
+      setForecasts,
+      setLocation,
+      searchText,
```

Comment:
There is an error popping up about `searchText` not being included in the dependency array, in this case the simple fix is to not pass it into the `getForecast` function at all, and instead use an empty string in its place.
We can do this in this case, as this `useEffect` is only being called on the initial render of the component and therefore the searchText will always be an empty string since that is what it is initialised to when the component first mounts. That will remove the error for you